README.md

```diff
@@ -179,12 +179,30 @@ To use the GitHub Action with Codacy integration:
 
 5.  Optionally, [enable the GitHub integration](https://docs.codacy.com/repositories-configure/integrations/github-integration/) on Codacy to have information about the analysis of the changed files directly on your pull requests.
 
-## Extra configurations
+## Parameters
 
-The Codacy GitHub Action is a wrapper for running the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli) and supports [the same parameters as the command `analyze`](https://github.com/codacy/codacy-analysis-cli#commands-and-configuration), with the following exceptions:
+The Codacy GitHub Action is a wrapper for running the [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli). For a list of supported input parameters, see [`action.yml`](./action.yml).
```

Comment:
I'd rather defer to the action's `yml` definition file here. We already link to the underlying analysis CLI in the same paragraph.

Reply:
That's a good idea because that's the simplest way of:

* Ensuring that users always see the updated list of allowed parameters
* We don't have to repeat the descriptions and ensure they're in sync :grimacing: